xcube_geodb/core/geodb.py

```diff
@@ -2072,7 +2073,7 @@ def _get_full_url(self, path: str) -> str:
             server_url = self._gs_server_url
             server_port = self._gs_server_port
 
-        if self._use_winchester and "geodb_geoserver" in path:
+        if self._use_winchester_gs and "geodb_geoserver" in path:
```

Comment:
Btw, underscores should be avoided in URL components, see https://stackoverflow.com/questions/119312/urls-dash-vs-underscore